frontend/finance-tracker/src/components/navigation/app-breadcrumb.tsx

```
@@ -0,0 +1,56 @@
+    const segments = pathname.split("/").filter((segment) => segment);
+    return (
+        <Breadcrumb>
+            <BreadcrumbList>
+                {segments.length === 0 ? (
+                    <BreadcrumbItem>
+                        <BreadcrumbPage>Root</BreadcrumbPage>
+                    </BreadcrumbItem>
+                ) : (
+                    segments.map((segment, index) => {
+                        const href =
+                            "/" + segments.slice(0, index + 1).join("/");
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)